tensorboard/webapp/notification_center/_views/notification_center_component.ts

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Notification} from './view_types';
+
+@Component({
+  selector: 'notification-center-component',
+  templateUrl: './notification_center_component.ng.html',
+  styleUrls: ['./notification_center_component.css'],
+})
+export class NotificationCenterComponent {
+  @Input() notifications!: Notification[];
+  @Output() onNotificationChanged = new EventEmitter<Boolean>();
```

Comment:
two things:
1. type it with `boolean`, not `Boolean`. Boolean is a constructor and there is a small differences but `boolean` is what you want.
2. would you not want to propagate _which_ notification got clicked on in case there are more than one?

Reply:
Thanks for the comment!
I think both the icon clicked and the notification click needed to be handled.
but to close the tab is on the icon click.
the click on the notification chunk depends on the buttons (`see more/less` and `see full`)
only `see full` leads to the tab close

for now I remove all output here in this pr and leave the functionality review to follow-up